config/software/python.rb

```diff
@@ -40,10 +40,12 @@
                       "--enable-universalsdk=/",
                       "--prefix=#{install_dir}/embedded"]
 
-  if ohai["platform_family"] == "mac_os_x"
+  if mac_os_x?
```

Comment:
should be `osx?` I think: https://github.com/DataDog/dd-agent-omnibus/blob/5.22.3/lib/ostools.rb#L16-L18 & https://github.com/DataDog/datadog-agent/blob/6.2.1/omnibus/lib/ostools.rb#L20-L22

Reply:
we have this in multiple recipes though...

Reply:
ok this one is actually provided by `chef-sugar`

Reply:
in the future we should probably get rid of our `lib/ostools.rb` and use the sugar-provided stuff directly instead